packages/lib/RotatingLogs.ts

```diff
@@ -0,0 +1,73 @@
+const moment = require('moment');
```

Comment:
We would like to remove the `moment` package eventually, as it's very large and pretty much deprecated. Could you use `dayjs` instead please? You can see in the server package how it's used